sdk/ai/azure-ai-assistants/tests/test_model_assistants_client.py

```diff
@@ -0,0 +1,706 @@
+# originally copied from azure-ai-inference 
+

```

Comment:
filename could be just test_assistants_client.py, no need to have model in the name